src/TwigComponent/src/CVA.php

```diff
@@ -0,0 +1,99 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\UX\TwigComponent;
+
+/**
+ * @author Mathéo Daninos <matheo.daninos@gmail.com>
+ */
+class CVA
```

Comment:
Final and/or experimental here ? Or this will be under BC promise from day one :|